docs/src/lang/records.md

```diff
@@ -148,17 +145,18 @@ is picked with `\E r \in [ field_1: S_1, ..., field_n: S_n]`.
 **Example in TLA+:**
 
 ```tla
-  [ name: { "A", "B", "C" }, port: 1..65535 ]
+  [ name: { "A", "B", "C" }, port: 65533..65535 ]
     \* A set of records. Each has two fields:
     \* field "name" that has the value from the set { "A", "B", "C" }, and
-    \* field "port" that has the value from the set 1..65535.
+    \* field "port" that has the value from the set 65533..65535.
 ```
 
-**Example in Python:** TLA+ functions are immutable, so we are using [frozendict]:
+**Example in Python:** We use a list instead of a set, since Python dictionaries are not hashable.
```

Comment:
This can be changed to saying we need `frozendict` because of the need for hashability.

Reply:
The change is now unrelated to frozendict.